### PR TITLE
refactor: move listinggrid to its own file

### PR DESCRIPTION
**Pull request by cbachmeier**

### CODE REVIEW

File: src/graphql/data/nft/NftBalance.ts [hunk 1]
```diff
@@ -159,7 +159,7 @@ export function useNftBalanceQuery(
       last,
       before,
     },
-    { fetchPolicy: 'store-or-network' }
+    { fetchPolicy: 'network-only' }
```
Comment (vm) [inline src/graphql/data/nft/NftBalance.ts:162]: unrelated?

**Review by vm — APPROVED** [review]
idk if the fetch policy is intended but ideally can rm for this pr

File: src/graphql/data/nft/NftBalance.ts (hunk 1 above)
Comment (cbachmeier) [reply src/graphql/data/nft/NftBalance.ts:162]: good catch
